### PR TITLE
Fix docs and add contributing guide

### PR DESCRIPTION
**Pull request by samutamm**
- Fix typo in docs
- Add CONTRIBUTING.md guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+
+# Contributing
+
+Welcome to the Easegress contributing guide. Are you looking for help or a way to create bug tickets or discuss new potential features? Perhaps you want to fix a typo in the documentation or are you interested in getting involved in the project? You can find answers to these questions and how to contribute to Easegress in this document.
+- [Contributing](#contributing)
+  - [Getting help](#getting-help)
+  - [Feature requests](#feature-requests)
+  - [Reporting general issues](#reporting-general-issues)
+  - [Contributing](#contributing-1)
+  - [Pull request guide](#pull-request-guide)
```
Comment (localvar) [inline CONTRIBUTING.md:10]:
```suggestion
# Contributing

Welcome to the Easegress contributing guide. Are you looking for help or a way to create bug tickets or discuss new potential features? Perhaps you want to fix a typo in the documentation or are you interested in getting involved in the project? You can find answers to these questions and how to contribute to Easegress in this document.

- [Contributing](#contributing)
  - [Getting help](#getting-help)
  - [Feature requests](#feature-requests)
  - [Reporting general issues](#reporting-general-issues)
  - [Contributing](#contributing-1)
  - [Pull request guide](#pull-request-guide)

```

**Review by suchen-sci — COMMENTED** [review]
add CONTRIBYTING.md to README.md?

**Review by haoel — APPROVED** [review]
Two optional comments 
1)  I think we can add a contact-us link (our slack) if people need the help instantly. 
2) add this document link to README.md

**Comment by samutamm** [comment]
>     1. I think we can add a contact-us link (our slack) if people need the help instantly.
There's already link to README.md part that contains the Slack link. I think it's better to have the link in one place so it's easier to keep up-to-date

**Review by suchen-sci — APPROVED** [review]

**Review by localvar — APPROVED** [review]

**Pull request merged**